src/content/docs/guides/integrating-with-fly-postgres.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Integrating a project with fly postgres
+description: A guide about Integrating a project with flycd postgres.
+---
```

Comment:
I would like to have a different title in the sidebar. See this link for [reference](https://starlight.astro.build/reference/frontmatter/#sidebar).

```suggestion
sidebar:
  label: Use Fly Postgres DB
---
```